vault/activity_log_testing_util.go

```diff
@@ -164,3 +164,10 @@ func (a *ActivityLog) SetEnable(enabled bool) {
 	defer a.fragmentLock.Unlock()
 	a.enabled = enabled
 }
+
+// GetEnabled returns the enabled flag on an activity log
+func (a *ActivityLog) GetEnabled() bool {
+	a.fragmentLock.RLock()
+	defer a.fragmentLock.RUnlock()
+	return a.enabled
+}
```

Comment:
Heh... I told Bez we should just add this so we could check GetEnabled() before and after SetConfig.

Reply:
yeah so this exists in an _ent file but not OSS so I just moved it over